SevenZip/ArchiveExtractCallback.cs

```diff
@@ -564,6 +564,11 @@ private static void ValidateFileNameAndCreateDirectory(string fileName)
         /// <returns></returns>
         private static string RemoveIllegalCharacters(string str, bool isDirectory = false)
         {
+#if NET462
+            var isUncPath = str.StartsWith(@"\\?\");
+            if(isUncPath)
+                str = str.Substring(@"\\?\".Length);
```

Comment:
Also only preference, but I'd really prefer

```
if (isUncPath)
{
    str = str.Substring(@"\\?\".Length);
}
```

Applies to more places in this PR.

Reply:
fixed